### PR TITLE
User Access Token resource

### PR DESCRIPTION
**Pull request by solankisamir**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/1213)

### CODE REVIEW

File: arm-apimanagement/2016-10-10/swagger/apimusers.json [hunk 1]
```diff
@@ -490,6 +490,59 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/users/{uid}/token": {
+      "post": {
+        "tags": [
+          "UserToken"
+        ],
+        "operationId": "Users_GetSharedAccessToken",
```
Comment (vishrutshah) [inline arm-apimanagement/2016-10-10/swagger/apimusers.json]: `Noun` part of `operationId` should match tags by conventions. Please update `operationId` to `UserToken_GetSharedAccessToken` or change the `tags` to `Users`. Former on will create operation group name as `UserToken` while later one creates operation group name as `Users`. So make sure sure which one you prefer in SDK

File: arm-apimanagement/2016-10-10/swagger/apimusers.json [hunk 2]
```diff
@@ -648,6 +701,42 @@
...
+				"name": "KeyTypeContract",
+				"modelAsString": false
+			  }
+			},
+		    "expiry": {
+			  "type": "string",
+			  "format": "date-time",
+			  "description": "The Expiry time of the Token. Maximum token expiry time is set to 30 days. The date conforms to the following format: `yyyy-MM-ddTHH:mm:ssZ` as specified by the ISO 8601 standard.\n"
+			}
+     	  },
```
Comment (vishrutshah) [inline arm-apimanagement/2016-10-10/swagger/apimusers.json]: please format the json file

File: arm-apimanagement/2016-10-10/swagger/apimusers.json [hunk 3]
```diff
@@ -490,6 +490,59 @@
...
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/users/{uid}/token": {
+      "post": {
+        "tags": [
+          "UserToken"
+        ],
+        "operationId": "Users_GetSharedAccessToken",
+        "description": "Gets the Shared Access Authorization Token for the User.",
```
Comment (vishrutshah) [inline arm-apimanagement/2016-10-10/swagger/apimusers.json]: please use keywords consitantly either "SharedAccessAuthorization" or "Shared Access Authorization"

Example: here v/s line 735

File: arm-apimanagement/2016-10-10/examples/ApiManagementUsersGetToken.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2016-10-10",
+    "subscriptionId": "subid",
+    "uid":"57f2af53bb17172280f44057",
+    "parameters": {
+         "keyType": "primary",
+         "expiry": "2017-05-09 18:20:08.7803554"
```
Comment (vishrutshah) [inline arm-apimanagement/2016-10-10/examples/ApiManagementUsersGetToken.json]: Does this follow the ISO8601 format of `yyyy-MM-ddTHH:mm:ssZ`?

Comment (vishrutshah) [inline arm-apimanagement/2016-10-10/examples/ApiManagementUsersGetToken.json]: Please see CI failure https://travis-ci.org/Azure/azure-rest-api-specs/jobs/230842021#L1107

**Review by vishrutshah — CHANGES_REQUESTED** [review]
@solankisamir Changes LGTM overall. Just left few comments to be addressed. Thanks!

**Comment by solankisamir** [comment]
@vishrutshah can you take a look at the changes?

**Comment by vishrutshah** [comment]
Changes looks good to me.

@ravbhatnagar Just wanted to make sure that "_Adding new operations into existing RP's existing api-version is considered to be an additive change from API Review Board perspective._" Please confirm. Thanks!

**Comment by vishrutshah** [comment]
Verified with @ravbhatnagar offline that adding new operations is not a breaking change in the RPC perspective. 

Thanks @solankisamir for addressing the comments. LGTM!

**Review by vishrutshah — APPROVED** [review]

**Pull request merged**